pkg/trace/watchdog/cpu.go

```diff
@@ -8,16 +8,19 @@
 
 package watchdog
 
-import "github.com/shirou/gopsutil/v3/process"
+import (
+	"errors"
+
+	"github.com/DataDog/gopsutil/cpu"
+)
 
 func cpuTimeUser(pid int32) (float64, error) {
-	p, err := process.NewProcess(pid)
+	times, err := cpu.Times(false)
 	if err != nil {
 		return 0, err
 	}
-	times, err := p.Times()
-	if err != nil {
-		return 0, err
+	if len(times) == 0 {
+		return 0, errors.New("no CPU times returned. Will report 0 CPU usage.")
```

Comment:
super-nit: capitalize the initial `no` since it's a full sentence

Reply:
Actually the idiom is to keep errors lowercased. The reason is that almost 99% of the time when logging an error you do something like:
```go
log.Error("Error doing something: %v", err)
```
... and then if the error is capitalised, it looks wrong.

Reply:
Yep, just noticed, thanks to your comment!